app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+<!--?xml version="1.0" encoding="UTF-8"?-->
 <resources>
-    <color name="colorPrimary">#008577</color>
-    <color name="colorPrimaryDark">#00574B</color>
-    <color name="colorAccent">#D81B60</color>
-</resources>
+    <color name="primaryColor">#43a047</color>
```

Comment:
Material design specifies that we should use 3 colors: colorPrimary, colorPrimaryDark (for toolbar) and colorAccent (views). It's a kind of standard for all developers.